src/styles.css

```diff
@@ -4,6 +4,19 @@
   box-sizing: border-box;
 }
 
+body {
+  background-image: url(img/mario.webp);
+  background-repeat: no-repeat;
+  background-size: cover;
+  font-family: 'Press Start 2P', cursive;
+  color: white;
+  text-shadow:
+    -1px -1px 0 black,
+    1px -1px 0 black,
+    -1px  1px 0 black,
+    1px  1px 0 black;
+}
+
```

Comment:
you did well with your project and it's a lovely design you have here, please endeavour to follow the given framework at all times, on the framework, the data postedd from the API should appear like this 👍 

Framework given :
![image](https://github.com/bohaz/Leaderboard/assets/20537712/c030b6ab-43b0-48a2-bfa5-318b657110db)

Your design :
![image](https://github.com/bohaz/Leaderboard/assets/20537712/0e4b209e-6c15-44e5-8839-fcc6f1f4f60f)

Reply:
Thanks! 🙌😊 I appreciate your professional review.